argocd_application_sets/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -11,19 +11,19 @@ include::snippets/technology-preview.adoc[]
 
 By using application sets, you can automate and manage the deployments of multiple Argo CD applications declaratively from a single mono-repository to many clusters at once with greater flexibility.
 
-With {gitops-title} 1.12 and later, you can manage the `ApplicationSet` resources in non-control plane namespaces by explicitly enabling and configuring the `ArgoCD` and `ApplicationSet` custom resources (CRs) as per your requirements. This functionality is particularly useful in multitenancy environments when you want to manage deployments of Argo CD applications for your isolated teams.
+With {gitops-title} 1.12 and later, as a cluster administrator, you can create and manage the `ApplicationSet` resources in non-control plane namespaces declaratively, other than the `openshift-gitops` control plane namespace,  by explicitly enabling and configuring the `ArgoCD` and `ApplicationSet` custom resources (CRs) as per your requirements. This functionality is particularly useful in multitenancy environments when you want to manage deployments of Argo CD applications for your isolated teams. This functionality is called the _ApplicationSet in any namespace_ feature in the Argo CD open source project.
 
 [NOTE]
 ====
 The generated Argo CD applications can create resources in any non-control plane namespace. However, the application itself will be in the same namespace as the application set resources.
 ====
 
-[id="prerequisites_enabling-app-sets-in-any-non-control-plane-namespaces"]
+[id="prerequisites_managing-app-sets-in-non-control-plane-namespaces_{context}"]
 == Prerequisites
```

Comment:
We have mentioned the exact content in id and then, we are agin using the context variable. Why?

See here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies 

Anchor id should be in this format:
[id="prerequisites_{context}"]
== Prerequisites